cmd/server.go

```diff
@@ -29,9 +29,12 @@ const (
 	serverStopAction
 	serverShutdownAction
 	serverDeleteAction
+	serverCreateAction
 )
 
 var forceFlag bool
+var servernameFlag, templateFlag, passwordFlag, hostnameFlag string
+var cpuFlag, memFlag, storageFlag int
```

Comment:
Use more idiomatic var-parantheses declaration here:
```go
var (
    # …
)
```

Event better, use a struct for this whereas fields are used for the individual flags, e.g., something along the way of

```go
type CreateOptions struct {
    Memory        int
    PlainPassword string
    #…
}

# …and then

func (f *foo) CreateOptions() CreateOptions {
	return CreateOptions{
		Memory:        f.Memory,
		PlainPassword: f.PlainPassword,
	}
}

# …

createCmd.PersistentFlags().IntVarP(&createOptions.Memory, "mem", "m", 1, "Memory (GB)")
```

Reply:
I like the idea of a structure in this context, thank you. Should I embed a mutex in the structure to make it more thread-safe?